defaults/main.yml

```diff
@@ -8,15 +8,16 @@ lin_rclone_repo_owner: rclone
 lin_rclone_binary: /usr/bin/rclone
 lin_rclone_config: /etc/rclone/rclone.conf
 
-lin_rclone_allow_nonroot: no
+lin_rclone_allow_nonroot: yes
 lin_rclone_group_gid: 911
 
 lin_rclone_idle_timeout: 900
 lin_rclone_mount_timeout: 10
-lin_rclone_vfs_cache_mode: writes
-lin_rclone_dir_cache_time: 30s
-
+lin_rclone_vfs_cache_mode: minimal
+lin_rclone_dir_cache_time: 1h
+#lin_rclone_rcaddr: 127.0.0.1:58888
```

Comment:
better rewrite this as
```
lin_rclone_rcaddr: ~ # example:  127.0.0.1:58888
```

Reply:
however, since you don't use it later on, you can just remove it

Reply:
this default is useless. if you define it but skip parameter on a mount you'll get port collision error between rclone copies